Dockerfile

```diff
@@ -1,28 +1,36 @@
-FROM golang:1.10.1-alpine3.7
+FROM golang:1.10.4-alpine3.7
 
 LABEL maintainer="Minio Inc <dev@minio.io>"
 
 ENV GOPATH /go
-ENV PATH $PATH:$GOPATH/bin
 ENV CGO_ENABLED 0
+
+WORKDIR /go/src/github.com/minio/
+
+RUN  \
+     apk add --no-cache ca-certificates 'curl>7.61.0' git && \
+     echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
```

Comment:
This line is also unnecessary. We can just `go get` and then `go build/install`. Any particular reasons for the other packages? 

This is a `golang` image.